src/content.tsx

```diff
@@ -7,9 +7,17 @@ import { storage } from 'webextension-polyfill'
 import { Tako, TakoProvider } from './components/Tako'
 import { TokenPrompt } from './components/TokenPrompt'
 import { useStore } from './store'
-import { waitForElement } from './waitForElement'
+import { onElementRemoval, waitForElement } from './waitForElement'
 
 const start = async () => {
+  if (!isRepoRoot() || !isRepoTree()) {
+    return
+  }
+
+  onElementRemoval('.tako', () => {
+    console.log('remount tako')
+    start()
+  })
```

Comment:
After the rendering, we could add a check for the element

```
  await waitForElement('.tako').catch(() => {
    start()
  })
```

Reply:
Not sure if that's necessary because rendering itself never fails. Rather, we render, then GH blows away the changes. By starting the removal check before we render, we never need to check whether render worked.